packages/terra-alert/src/Alert.jsx

```diff
@@ -107,28 +116,29 @@ const Alert = ({
   type,
   ...customProps
 }) => {
+  const theme = React.useContext(ThemeContext);
   const [isNarrow, setIsNarrow] = useState();
+
   const defaultTitle = type === AlertTypes.CUSTOM ? '' : <FormattedMessage id={`Terra.alert.${type}`} />;
-  const attributes = { ...customProps };
-  const alertClassNames = cx([
-    'alert-base',
-    type,
-    { narrow: isNarrow },
-    { wide: !isNarrow },
-    attributes.className,
-    { [`${customColorClass}`]: type === AlertTypes.CUSTOM },
+  const alertClassNames = classNames([
+    cx([
```

Comment:
The array brackets for classnames is optional. This syntax is fine, I just wanted to throw that out there for future reference. Classnames and cx will both spread arguments as an array internally. 

```js
cx(['one', 'two', 'three']) === cx('one', 'two', 'three')
```

Reply:
Good point. I'll simplify now